docs/index.md

```diff
@@ -34,8 +34,7 @@ terraform {
 }
 
 provider "cloudflare" {
-  email   = var.cloudflare_email
```

Comment:
you shouldn't manually make changes to this file, see https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md

Reply:
fixed!